### DISCUSSION TITLE
Section title as a link

### DISCUSSION THREAD

**Discussion by markpiller** [squidfunk/mkdocs-material#7281] [Q&A] [answered]
Hello,

I am trying to figure how to implement the same TOC behavior as shown in the Material for MlkDocs documentation. Specifically here:

https://squidfunk.github.io/mkdocs-material/tutorials/

I see the `Tutorials` section is configured this way:
```
    - Tutorials:
      - tutorials/index.md
      - "Blogs":
        - tutorials/blogs/basic.md
        - tutorials/blogs/navigation.md
        - tutorials/blogs/engage.md
  ```
I am doing the same thing in mine:
```
- Configuration:
  - configuration/about-configuration.md
  - 'Configuration Properties':
    - 'Developer': 'configuration/developer.md'
```
However, it renders differently - no matter what I do, the `About Configuration` node is added to the TOC:
 
<img width="167" alt="Backendless Pro Administrator Guide 2024-06-19 15-40-42" src="https://github.com/squidfunk/mkdocs-material/assets/120929/9a6118e1-c3c9-4ee5-a7cd-3b809c3483d3">

Please help.

Mark

**Comment by alexvoss** [comment, marked as answer]
Hi, you probably just need to add configuration for [section index pages](https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#section-index-pages) to your `mkdocs.yml`.

**Comment by markpiller** [comment]
That was it! Thank you @alexvoss !
